src/base-styles/grid.scss

```diff
@@ -1,5 +1,10 @@
 $column_count: 12;
 
+$mobile_size: 375px; // legacy breakpoint size for mobile
+$tablet: map.get($breakpoints, "m");
+$desktop-small: map.get($breakpoints, "l");
+$desktop-big: map.get($breakpoints, "xl");
```

Comment:
`grid.scss` makes extensive use of breakpoints that are not all mobile first, so this maps values from the new breakpoints map to the legacy sass vars